File: multimediaBack-end/multimedia/src/main/java/ao/isptec/multimedia/dto/GrupoMembrosGrupo.java
package ao.isptec.multimedia.dto;

import java.util.List;

import ao.isptec.multimedia.model.Grupo;
import ao.isptec.multimedia.model.MembroGrupo;

public class GrupoMembrosGrupo {

    private Grupo grupo;
    private List<MembroGrupo> membrosGrupo;

    public GrupoMembrosGrupo(Grupo grupo, List<MembroGrupo> membrosGrupo) {
        this.grupo = grupo;
        this.membrosGrupo = membrosGrupo;
    }

    public Grupo getGrupo() {
        return grupo;
    }

    public void setGrupo(Grupo grupo) {
        this.grupo = grupo;
    }

    public List<MembroGrupo> getMembrosGrupo() {
        return membrosGrupo;
    }

    public void setMembrosGrupo(List<MembroGrupo> membrosGrupo) {
        this.membrosGrupo = membrosGrupo;
    }

}

File: isp-media-angular/src/app/models/GrupoMembrosGrupo.ts
import { Grupo } from "./Grupo";
import { MembroGrupo } from "./MembroGrupo";
import { Utilizador } from "./Utilizador";

export class GrupoMembrosGrupo {
    constructor(
        public grupo: Grupo,
        public membrosGrupo: Array<MembroGrupo>
    ) { }
}